mk/package-and-sign.sh

```diff
@@ -39,7 +39,8 @@ REPO=${XENADMIN_DIR}
 OUTPUT_DIR=${ROOT}/output
 DOTNETINST=${REPO}/dotNetInstaller
 	
-source ${REPO}/Branding/branding.sh
+BRANDING_BRAND_CONSOLE=[XenCenter]
```

Comment:
Hi Gabor
Yes, it is not a hard code value, it is with [] which will be rebranded when build in rebranding.sh